src/framebuffer/mod.rs

```diff
@@ -57,13 +59,31 @@ pub trait FramebufferDraw {
         rad: u32,
         c: common::color,
     ) -> common::mxcfb_rect;
+    /// Draws a polygon
+    fn draw_polygon(
+        &mut self,
+        &[cgmath::Point2<i32>],
+        fill: bool,
+        c: common::color,
+    ) -> common::mxcfb_rect;
     /// Draws a bezier curve begining at `startpt`, with control point `ctrlpt`, ending at `endpt` with `color`
     fn draw_bezier(
         &mut self,
         startpt: cgmath::Point2<f32>,
         ctrlpt: cgmath::Point2<f32>,
         endpt: cgmath::Point2<f32>,
         width: f32,
+        samples: i32,
+        v: common::color,
+    ) -> common::mxcfb_rect;
+    /// Draws a bezier curve begining at `startpt`, with control point `ctrlpt`, ending at `endpt`
+    /// with a width at each point and color `color`
```

Comment:
After seeing the implementation in the other file for `draw_dynamic_bezier`, I was going to say let's make sure we document how it allows individual widths for each point but looks like you've already taken care of that. :+1: